frontend/src/scenes/dashboard/SharedDashboard.js

```diff
@@ -0,0 +1,29 @@
+import './../../style.scss'
+import './DashboardItems.scss'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Provider } from 'react-redux'
+import { getContext } from 'kea'
+
+import { initKea } from '~/initKea'
+import { Dashboard } from './Dashboard'
+
+initKea()
+
+let dashboard = window.__SHARED_DASHBOARD__
+ReactDOM.render(
+    <Provider store={getContext().store}>
+        <div style={{ background: 'var(--gray-background)', minHeight: '100vh', top: 0, padding: '2rem' }}>
+            <div>
+                <h1 style={{ display: 'inline-block' }}>{dashboard.name}</h1>
+                <h1 style={{ float: 'right' }}>{dashboard.team_name}</h1>
+            </div>
```

Comment:
This is good enough for now, but ideally this could be done with flexbox to avoid issues like this:

![image](https://user-images.githubusercontent.com/53387/86571291-7ac28700-bf71-11ea-9482-befe7dd11142.png)